packages/core/core/src/serializer.js

```diff
@@ -161,6 +161,10 @@ function mapObject(object: any, fn: (val: any) => any, preOrder = false): any {
 }
 
 export function prepareForSerialization(object: any): any {
+  if (object?.$$raw) {
+    return object;
```

Comment:
Previously, we seem to have deleted that property (in this function, a few lines further down). I hope that's not a problem now.

I guess this also enables an optimization in `restoreDeserializedObject`: to skip the mapping if `$$raw: true` is set?

Reply:
I think it only moves the `$$raw` property to the wrapper object that is created after.